src/index.js

```diff
@@ -10,6 +9,7 @@ import {
   javascript,
   version as webpackVersion,
 } from 'webpack';
+import createHash from 'webpack/lib/util/createHash';
```

Comment:
Please use `import { utils: { createHash } } from 'webpack'`, no need hardcore path to exported utils

Reply:
Can't destructure in an import statement.

Reply:
Just use `webpack.utils.createHash`

Reply:
Doesn't work.

Reply:
Strange, I will look at this in near future, anyway thanks for the PR